vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -215,7 +217,20 @@ def complete_warmup(self):
         """Turn off warmup mode once the warmup is complete"""
         self.warmup_mode = False
 
-    def _update_states(self, scheduler_output: SchedulerOutput):
+    def build_attn_metadata(self,
+                            _: ModelForwardInputs) -> SpyreAttentionMetadata:
+
+        # TODO: probably sooner we will need
+        attn_metadata = SpyreAttentionMetadata()
+        return attn_metadata
```

Comment:
Do we need to have this function defined in the superclass as well if we only need to use it for continuous batching?

Reply:
Same question for the below two functions as well

Reply:
Oh, now I get it. Static batching uses the default implementation defined in the base class. 🤦